### ISSUE TITLE
Not work with Jodit.BR

### ISSUE THREAD

**Issue by Kirilych** [xdan/jodit#7]
Not work Jodit.**BR** (enter) mode

**Comment by xdan** [comment]
Please, show you init code

**Comment by Kirilych** [comment]
sorry, editor work, but not correct

```
<textarea id="text_area"> <img src="tigr.jpg" style="width:100px;"/> </textarea>
<script>
	var object = document.getElementById("text_area");
	new Jodit(object, {
		enter: Jodit.BR
	});
</script>
```
without brakes:
![1](https://user-images.githubusercontent.com/5024436/33238343-71cc33fc-d294-11e7-8695-7a1acd89205c.png)

with 3 brakes:
![2](https://user-images.githubusercontent.com/5024436/33238345-76ef9b44-d294-11e7-93da-cd10521bc1a7.png)

**Comment by xdan** [comment]
Hi, thank you. Fixed in 3.0.36
Can i close it?
